Misc/NEWS.d/next/Library/2024-01-21-16-32-55.gh-issue-114257.bCFld5.rst

```diff
@@ -0,0 +1 @@
+Ignore the :exc:`FileNotFound` error in :func:`ctypes.util._is_elf` and just return ``False``.
```

Comment:
`_is_elf()` is a private function, the end user knows nothing about it. Please describe the change in the terms of public API.